src/wallet/wallet.ts

```diff
@@ -126,14 +126,16 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     network,
     passphrase = '',
     enableWs = true,
+    storage = null,
   }: {
     requestPassword: Function,
     seed?: string | null,
     xpriv?: string | null,
     xpub?: string | null,
     network: Network,
     passphrase?: string,
-    enableWs: boolean,
+    enableWs?: boolean,
```

Comment:
If this is an optional boolean, could we consider the default value to be `false`?